src/test/java/Homework17A.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17A extends MyBaseTest{
+//    public static WebDriver driver = new ChromeDriver();
+//    we can remove this webdriver initialization
+
+
+    @Test
+    public void addSongtoPlaylist() throws InterruptedException {
+        //login("simplyd2d@gmail.com","te$t$tudent");
+    enterEmail("simplyd2d@gmail.com");
+    enterPassword("te$t$tudent");
+    clickSubmit();
+    Thread.sleep(2000);
+
+    searchSong("Take My Hand");
+    Thread.sleep(2000);
+    viewAllSearchResults();
+    selectSong();
+    clickAddToButton();
+    selectPlaylistname("David");
+
+       WebElement addedSongAlert = driver.findElement(By.cssSelector("div.success.show"));
+        Assert.assertTrue(addedSongAlert.isDisplayed());
+
+        //Assert.assertTrue(getNotificationText().contains("Added 1 song into"));
+        /* Assert.assertTrue(isNotificationPopUpPresent();   This I am unsure off? */
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._